internal/common/basic.go

```diff
@@ -157,7 +157,7 @@ func CheckWorkerNodesReady(wcClient *client.Client, values *application.ClusterV
 	}
 }
 
-func checkAllPodsSuccessfulPhase(wcClient *client.Client) func() error {
+func CheckAllPodsSuccessfulPhase(wcClient *client.Client) func() error {
```

Comment:
If we're re-using these it might be time to pull these helpers out into `clustertest`

Reply:
Yeah, it would be better probably, I just didn't want to make this a larger refactor in this PR.

Reply:
Yeah makes sense. Just thinking for future improvement. :)

Reply:
I was thinking about this these days, and if these checks are re-used just inside of cluster-test-suites, I think it's fine for them to stay there and be re-used within cluster-test-suites, since it's less PRs to make changes.

Having some test funcs like this (defined in cluster-test-suites) re-used in multiple suites inside of cluster-test-suites sounds fine to me.

OTOH if we would reuse them in other projects then clustertest makes more sense.